Failed to build with -DUSE_STEAMWEBRTC=ON and cmake 3.20

# Steps to reproduce
1. Use cmake version 3.20.3 (I have manjaro linux installed)
2. Install dependecies `sudo pacman -S openssl protobuf`
3. Clone GameNetworkingSockets with `git clone --recursive https://github.com/ValveSoftware/GameNetworkingSockets`
4. Try to compile with p2p 
```
mkdir build && cd build
cmake -DUSE_STEAMWEBRTC=ON ..
```

# Error
```
-- Crypto library for AES/SHA256: OpenSSL
-- Crypto library for ed25519/curve25519: OpenSSL
-- Link-time optimization: OFF
-- Configuring done
CMake Error: install(EXPORT "GameNetworkingSockets" ...) includes target "GameNetworkingSockets" which requires target "steamwebrtc" that is not in any export set.
CMake Error: install(EXPORT "GameNetworkingSockets" ...) includes target "GameNetworkingSockets_s" which requires target "steamwebrtc" that is not in any export set.
-- Generating done
CMake Generate step failed.  Build files cannot be regenerated correctly.
```

# Full log
```
-- The C compiler identification is GNU 11.1.0
-- The CXX compiler identification is GNU 11.1.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/lib/ccache/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/lib/ccache/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Setting CMAKE_BUILD_TYPE to 'RelWithDebInfo' as none was specified.
-- Found OpenSSL: /usr/lib/libcrypto.so (found version "1.1.1k")  
-- OPENSSL_INCLUDE_DIR = /usr/include
-- Looking for EVP_MD_CTX_free
-- Looking for EVP_MD_CTX_free - found
-- Looking for EVP_PKEY_get_raw_public_key
-- Looking for EVP_PKEY_get_raw_public_key - found
-- Found Protobuf: /usr/lib/libprotobuf.so;-lpthread (found version "3.16.0") 
-- Found Threads: TRUE  
-- Looking for sys/types.h
-- Looking for sys/types.h - found
-- Looking for stdint.h
-- Looking for stdint.h - found
-- Looking for stddef.h
-- Looking for stddef.h - found
-- Check size of void*
-- Check size of void* - done
-- Performing Test _fvisibility_hidden_TEST
-- Performing Test _fvisibility_hidden_TEST - Success
-- Performing Test _fno_strict_aliasing_TEST
-- Performing Test _fno_strict_aliasing_TEST - Success
-- Performing Test _Wall_TEST
-- Performing Test _Wall_TEST - Success
-- Performing Test _Wno_unknown_pragmas_TEST
-- Performing Test _Wno_unknown_pragmas_TEST - Success
-- Performing Test _Wno_sign_compare_TEST
-- Performing Test _Wno_sign_compare_TEST - Success
-- Performing Test _Wno_unused_local_typedef_TEST
-- Performing Test _Wno_unused_local_typedef_TEST - Success
-- Performing Test _Wno_unused_const_variable_TEST
-- Performing Test _Wno_unused_const_variable_TEST - Success
-- Performing Test _Wno_nested_anon_types_TEST
-- Performing Test _Wno_nested_anon_types_TEST - Success
-- Performing Test _Wno_format_truncation_TEST
-- Performing Test _Wno_format_truncation_TEST - Success
-- Performing Test _Wstrict_prototypes_TEST
-- Performing Test _Wstrict_prototypes_TEST - Success
-- Performing Test _fvisibility_inlines_hidden_TEST
-- Performing Test _fvisibility_inlines_hidden_TEST - Success
-- Performing Test _Wno_reorder_TEST
-- Performing Test _Wno_reorder_TEST - Success
-- Performing Test _Wno_non_virtual_dtor_TEST
-- Performing Test _Wno_non_virtual_dtor_TEST - Success
-- Performing Test _Wno_zero_as_null_pointer_constant_TEST
-- Performing Test _Wno_zero_as_null_pointer_constant_TEST - Success
-- Performing Test _fno_exceptions_TEST
-- Performing Test _fno_exceptions_TEST - Success
-- Performing Test _fno_rtti_TEST
-- Performing Test _fno_rtti_TEST - Success
-- Crypto library for AES/SHA256: OpenSSL
-- Crypto library for ed25519/curve25519: OpenSSL
-- Link-time optimization: OFF
-- Configuring done
CMake Error: install(EXPORT "GameNetworkingSockets" ...) includes target "GameNetworkingSockets" which requires target "steamwebrtc" that is not in any export set.
CMake Error: install(EXPORT "GameNetworkingSockets" ...) includes target "GameNetworkingSockets_s" which requires target "steamwebrtc" that is not in any export set.
-- Generating done
CMake Generate step failed.  Build files cannot be regenerated correctly.
```

--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -293,8 +293,8 @@ add_library(GameNetworkingSockets::static ALIAS GameNetworkingSockets_s)
 target_compile_definitions(GameNetworkingSockets_s INTERFACE STEAMNETWORKINGSOCKETS_STATIC_LINK)
 gamenetworkingsockets_common(GameNetworkingSockets_s)
 
-# Install rules
 
+# Install rules
 install(
 	TARGETS 
 		GameNetworkingSockets
@@ -307,8 +307,9 @@ install(
 
 install(DIRECTORY ../include/ DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/GameNetworkingSockets)
 
-# Export doesn't work if we're using WebRTC and the Abseil dependency came from the submodule
-if(NOT (USE_STEAMWEBRTC AND STEAMWEBRTC_ABSL_SOURCE STREQUAL submodule))
+# Export file required for find_package(GameNetworkingSockets CONFIG) to work
+# There's some issues around getting this to work properly when WebRTC is enabled.
+if(NOT (USE_STEAMWEBRTC))
 	install(
 		EXPORT GameNetworkingSockets
 		DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/GameNetworkingSockets

